{{cookiecutter.__src_folder_name}}/README.md

```diff
@@ -55,17 +55,33 @@ Run gunicorn on the app:
 
 ```sh
 {% if cookiecutter.project_backend == "flask" %}
-python3 -m gunicorn 'src.flaskapp:create_app()' -c src/gunicorn.conf.py
+python3 -m gunicorn 'src.flaskapp:create_app()' --reload
```

Comment:
For local dev, multiple workers is overkill, you just want 1 worker and auto-reload